### PR TITLE
Internal: Create a test for the minimalist link in bio widget [ED-16381]

### PR DESCRIPTION
**Pull request by MichaelLiamin**

### CODE REVIEW

File: tests/playwright/sanity/minimalist.test.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	await page.getByRole( 'button', { name: 'Insert Media' } )
+		.or( page.getByRole( 'button', { name: 'Select' } ) ).nth( 1 ).click();
+	await page.locator( '[data-collapse_id="bio_section"]' ).click();
+	await page.locator( '[data-setting="bio_heading"]' ).fill( 'This is a test' );
+	await page.locator( '[data-setting="bio_title"]' ).fill( 'This is a test' );
+	await page.locator( '[data-setting="bio_description"]' ).fill( 'This is a test' );
+	await editor.togglePreviewMode();
+	expect( await editor.getPreviewFrame()
+		.locator( '.e-link-in-bio__content' )
+		.screenshot( { type: 'jpeg', quality: 90 } ) )
```
Comment (hein-obox) [inline tests/playwright/sanity/minimalist.test.ts:32]: We can remove this, if we use a `png` file:

```suggestion
```

File: tests/playwright/sanity/minimalist.test.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		.toMatchSnapshot( 'minimalist.jpeg' );
```
Comment (hein-obox) [inline tests/playwright/sanity/minimalist.test.ts:33]:
```suggestion
		.toMatchSnapshot( 'minimalist.png' );
```

File: tests/playwright/sanity/minimalist.test.ts (hunk 1 above)
Comment (Svitlana-Dykun) [reply tests/playwright/sanity/minimalist.test.ts]: I would replace it with 
await expect( widget ).toHaveScreenshot( 'screenshotName.png' );

File: tests/playwright/sanity/minimalist.test.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+	// Assert.
+	await page.locator( '.elementor-control-media__preview' ).click();
+	await page.locator( 'text=Media Library' ).click();
+	await page.waitForSelector( 'text=Insert Media' );
+	await page.click( '[aria-label="mountain-image"], li[tabindex="0"]' );
+	await page.getByRole( 'button', { name: 'Insert Media' } )
+		.or( page.getByRole( 'button', { name: 'Select' } ) ).nth( 1 ).click();
+	await page.locator( '[data-collapse_id="bio_section"]' ).click();
+	await page.locator( '[data-setting="bio_heading"]' ).fill( 'This is a test' );
```
Comment (Svitlana-Dykun) [inline tests/playwright/sanity/minimalist.test.ts:26]: I suggest using different text values for input fields, to ensure they are working as expected (This is a test, This is my nice bio title etc)

File: tests/playwright/sanity/minimalist.test.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+	const page = await context.newPage();
+	const wpAdmin = new WpAdminPage( page, testInfo, apiRequests );
+	const editor = new EditorPage( page, testInfo );
+	await wpAdmin.openNewPage();
+	await editor.closeNavigatorIfOpen();
+
+	// Act.
+	await editor.addWidget( 'link-in-bio' );
+
+	// Assert.
```
Comment (Svitlana-Dykun) [inline tests/playwright/sanity/minimalist.test.ts:18]: As actual assertion done below, I would move this line

**Review by Svitlana-Dykun — CHANGES_REQUESTED** [review]

**Review by Svitlana-Dykun — APPROVED** [review]

**Pull request merged**